src/widgets/dialog.ts

```diff
@@ -31,7 +31,7 @@ export class Dialog extends Widget {
 			.element(by.buttonText(name)));
 	}
 
-	public getMesssagePopup(): MesssagePopup {
-		return new MesssagePopup();
+	public getMesssagePopup(): MessagePopup {
```

Comment:
You could drop the 3rd s here, also :)

Reply:
oops, sure I can... thanks for your review!!